cmd/avrogen/main.go

```diff
@@ -25,6 +25,8 @@ type config struct {
 	FullName    bool
 	Encoders    bool
 	FullSchema  bool
+	PlainMap    bool
```

Comment:
The decoder will not work for map like it does for slice currently.

Reply:
Thanks for your feedback, however I'm not sure what you are referring to here. What would need to change in the decoder for this to work?